arches_her/settings.py

```diff
@@ -197,18 +197,27 @@
 try:
     from .package_settings import *
 except ImportError:
-    pass
+    try: 
+        from package_settings import *
+    except ImportError as e:
+        pass
 
 try:
     from .settings_local import *
-except ImportError:
-    pass
+except ImportError as e:
+    try: 
+        from settings_local import *
+    except ImportError as e:
+        pass
 
 if DOCKER:
```

Comment:
what would happen here if there's no settings_local file with a DOCKER k/v ? Will it return falsy or throw a keyerror?

Reply:
It's fine if there's no settings_local, but it won't work in docker.  settings_local is required for it to load the settings_docker file with the db connection info.

Reply:
ah the wrong line got highlighted -- this is about the `if DOCKER:` line. I was able to repro a break by doing an `if FOO:` line on my local

Reply:
oh I missed the `DOCKER = False` line above, this is good!